lib/sycamore/sycamore/transforms/extract_graph_entities.py

```diff
@@ -15,6 +18,17 @@
 logger = logging.getLogger(__name__)
 
 
+ENTITY_DEFAULT_PROMPT = """
+    -Instructions-
+    You are a information extraction system.
+
+    You will be given a sequence of data in different formats(text, table, Section-header) in order.
+    Your job is to extract entities from the text input that match the entity schemas provided. Each entity
+    and property extracted should directly reference part of the text input provided.
+
+    """
```

Comment:
move to default prompts (something like `sycamore/llms/prompts/default_prompts.py`)

Reply:
I realized this, but the thing is all those prompts are guidance prompts. These have to be test cause currently guidance is not compatible with structured outputs(which is what this stack uses). Should I still move them over?

Reply:
yeah. they claim to be guidance prompts but really it's just an object with a particular string property. Someday we'll make a call on guidance and then it will be really nice to have all the prompts in one place